src/test/java/com/clouway/cuse/SearchEngineContractTest.java

```diff
@@ -3,29 +3,23 @@
 import com.clouway.cuse.gae.filters.SearchFilters;
 import com.clouway.cuse.spi.*;
 import com.google.appengine.labs.repackaged.com.google.common.collect.Lists;
-import com.google.appengine.tools.development.testing.LocalSearchServiceTestConfig;
-import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.labs.repackaged.com.google.common.collect.Sets;
```

Comment:
I didn't mean to remove them completely, just remove the `.repackaged` portion of the import, so it would be:
```java
import com.google.common.collect.Lists;
import com.google.common.collect.Sets;
```